lib/dl_connector_bundle_chs3/dl_connector_bundle_chs3/chs3_yadocs/core/lifecycle.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+from typing import ClassVar
+
+from dl_constants.enums import FileProcessingStatus
+from dl_core.connectors.base.lifecycle import ConnectionLifecycleManager
+from dl_core.reporting.notifications import get_notification_record
+from dl_core.utils import (
+    make_user_auth_cookies,
+    make_user_auth_headers,
+)
+
+from dl_connector_bundle_chs3.chs3_base.core.lifecycle import BaseFileS3ConnectionLifecycleManager
+from dl_connector_bundle_chs3.chs3_yadocs.core.constants import NOTIF_TYPE_YADOCS_STALE_DATA
+from dl_connector_bundle_chs3.chs3_yadocs.core.us_connection import YaDocsFileS3Connection
+
+
+class YaDocsFileS3ConnectionLifecycleManager(
```

Comment:
I remember that we agreed not to create a lifecycle manager before we bother about data update
This cannot work at the moment because the only data update file-uploader-api is aware of is for gsheets
But ok, let's keep it